src/test/java/PlaylistsPage.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PlaylistsPage extends BasePage{
+    public PlaylistsPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    //Locators
+    By renamePlaylistField = By.cssSelector("[name='name']");
+    By updatedPlaylistNotification = By.cssSelector(".success.show");
+    By firstPlaylist = By.xpath("//section[@id='playlists']//li[@class='playlist playlist'][1]");
+    //Methods
+    public void doubleClickOnPlaylist() {
+        doubleClick(firstPlaylist);
+    }
+    public void renamePlaylist(String playListName){
+        findElement(renamePlaylistField).sendKeys(Keys.chord(Keys.CONTROL, "A", Keys.BACK_SPACE));
+        findElement(renamePlaylistField).sendKeys(playListName);
+        findElement(renamePlaylistField).sendKeys(Keys.ENTER);
+    }
+    public String getUpdatedPlaylistNotification(){
+        return findElement(updatedPlaylistNotification).getText();
+    }
+
+    }
+
```

Comment:
A package in Java is used to group related classes. 

For example package "pages" - Includes the Page Objects (or Page Classes)

For example

![image](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/27e94b6d-4d57-4c2a-8022-e828ff1276f7)